README.md

```diff
@@ -34,3 +26,12 @@ Check out [our documentation](https://docs.tenzir.com/) where your find
 tutorials that walk your through the first steps, how-to guides to solve a
 specific problem, explanations of key concepts, and an in-depth reference of the
 nitty-gritty technical details.
+
+## Community
+
+<a href="https://discord.gg/xqbDgVTCxZ" alt="Tenzir Discord community">
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://invidget.switchblade.xyz/xqbDgVTCxZ">
+  <img alt="Tenzir Discord community" src="https://invidget.switchblade.xyz/xqbDgVTCxZ?theme=light">
```

Comment:
Is the hard coding of theme `light` on purpose here?

Reply:
It's both light and dark, this is the snippet recommended from upstream. See the `<source ...>` tag right above, which defines the dark mode image.